packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -44,9 +41,18 @@ NOTIFICATION_LIST_ITEM_SHOW_LESS=Less
 #XBUT: Tooltip text for 'Overflow' button in the NotificationListItem, that opens a menu with actions
 NOTIFICATION_LIST_ITEM_MENU_BTN_TITLE=Actions
 
+#XBUT: Label text for 'More' lin in the NotificationListItem, that shows the full text, if truncated
+NOTIFICATION_LIST_ITEM_MORE_LINK_LABEL_FULL=More button. Show the full texts
+
+#XBUT: Label text for 'More' lin in the NotificationListItem, that hides the truncated text
```

Comment:
I suggest:
Label text for 'More' link in the NotificationListItem, that truncates the text